### ISSUE TITLE
Carnalitas 1.2 - Character Fetish System

### ISSUE THREAD

**Issue by cherisong** [cherisong/Carnalitas#70]
- [ ] Come up with a list of fetishes
- [ ] Randomly seed characters with fetishes
- [ ] Add effects and triggers for modders to check, set, and remove fetishes
- [ ] Add GUI on character window to show fetishes

**Comment by Ernie0013** [comment]
A few years back a did some weird online test, luckily I still had the results lying around. Here's the list (ordered alphabetically):
[list-of-kinks.txt](https://github.com/cherisong/Carnalitas/files/5251096/list-of-kinks.txt)
Some of the things on here I don't even know what they are/mean.

**Comment by CaffeGit** [comment]
We can probably start by identifying nigh-mandatory inclusions, like (in no particular order):
- Foot play
- Bondage
- Sadism/masochism
- Domination/Submission
- Scat/watersports

Maybe you consider tentacles a mandatory inclusion, too, idk

It might also be worth trying to limit the number of two-sided kinks, like sadism/masochism, just to avoid trait spam.

**Comment by cherisong** [comment]
yeah, I was thinking of having a variable list full of flags, and having the widget pull from it

**Comment by CaffeGit** [comment]
[Source for fetish prevalence](https://www.nature.com/articles/3901547)

**Comment by CaffeGit** [comment]
[Source for most common fetishes](https://www.researchgate.net/publication/267627883_What_Exactly_Is_an_Unusual_Sexual_Fantasy)

**Comment by Ernie0013** [comment]
**Suggestion:**
Currently, fetishes are distributed randomly and equally. I think it'd be more realistic to give different fetishes different chances of being assigned. Like, scat isn't a common a fetish as, say, anal.
Also, maybe they should be related to traits. The obvious example is the `sadistic` trait having a much higher chance of having a sadist fetish (maybe even guaranteed).
